cves/kernel/CVE-2014-3183.yml

```diff
@@ -147,10 +167,23 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Bug (disclosed in https://bugs.chromium.org/p/project-zero/issues/detail?id=90)
+    appeared to be found around the same time as another bug (CVE-2014-3182,
+    disclosed in https://bugs.chromium.org/p/project-zero/issues/detail?id=89),
+    in the same driver (hid-logitech-dj.c). Both boths were similar in spirit
+    (incorrect bound checks leading to overflow)
+
+    Looking at https://www.openwall.com/lists/oss-security/2014/09/11/21 where the CVEs
+    were announced, all CVEs were about overflow vulnerabilities in HID drivers. I assume
+    the author found these bugs by manually audited Linux HID drivers and look for places
```

Comment:
Try to reword this sentence. It doesn't read correctly out loud, which hurts clarity and understanding

Reply:
I tried rewriting it a bit but I agree it still sounds "weird". If you have a better idea on how to rewrite it then I'd take it.